adam_core/observations/tests/test_ades.py

```diff
@@ -0,0 +1,235 @@
+from importlib.resources import files
+
+import pytest
+
+from ...time import Timestamp
+from ..ades import (
+    ADESObservations,
+    ObsContext,
+    ObservatoryObsContext,
+    SoftwareObsContext,
+    SubmitterObsContext,
+    TelescopeObsContext,
+)
+
+
+@pytest.fixture
+def ades_obscontext():
+    # Nearly real metadata used for ADAM::THOR observations
+    measurers = [
+        "J. Moeyens",
+        "M. Juric",
+        "S. Nelson",
+        "A. Koumjian",
+        "K. Kiker",
+        "N. Tellis",
+        "D. Veronese-Milin",
+        "A. Posner",
+        "E. Lu",
+        "C. Fiaschetti",
+        "D. Remy",
+    ]
+
+    software_context = SoftwareObsContext(
+        objectDetection="Asteroid Discovery Analysis and Mapping + Tracklet-less Heliocentric Orbit Recovery (ADAM::THOR)",
+    )
+
+    submitter_context = SubmitterObsContext(
+        name="J. Moeyens", institution="B612 Asteroid Institute"
+    )
+
+    fundingSource = "WilliamBowes, McGregorGirand, Tito's Vodka, PRawls, SKraus, Yishan/KWong, SGalitsky, Google"
+    comments = [
+        "THIS IS A TEST FILE CONTAINING FAKE OBSERVATIONS",
+        "Discovery candidates found by members of the Asteroid Institute, a program of B612 Foundation,",
+        "using the institute's ADAM::THOR discovery service running on Google Cloud.",
+    ]
+
+    obs_contexts = {
+        "W84": ObsContext(
```

Comment:
Instead of just for tests, should we also have some mini fixtures of the ObsContexts to have on hand for datasets we use regularly?

Reply:
Would it make sense to store them in a separate repo instead of in adam_core (maybe a private `adam-metadata`)?

Reply:
I can understand wanting that separation. Not sure if the overhead is worth it, I could go either way.